src/pluto/pluto.py

```diff
@@ -477,15 +477,16 @@ def generate_graph(v1, v2):
                     "firm07_flag",
                     "pfirm15_flag",
                 ]
-                x = [(v1[i] / v2[i] - 1) * 100 for i in y]
+                x = [((v2[i] - v1[i])/v1[i])*100  for i in y]
+                diff=[v2[i] - v1[i]  for i in y]
                 real_v1 = [v1[i] for i in y]
                 real_v2 = [v2[i] for i in y]
                 hovertemplate = "<b>%{x}</b> %{text}"
                 text = []
                 for n in range(len(y)):
                     text.append(
-                        "Percent Change: {:.2f}%<br>Prev: {} Current: {}".format(
-                            x[n],
+                        "Diff: {} Prev: {} Current: {}".format(
```

Comment:
One thing I just thought of is that I think it reads better if `Current` is first followed by `Prev`

Reply:
oh right, that will look better.